grpc/src/main/java/io/stargate/grpc/service/SchemaNotificationsHandler.java

```diff
@@ -66,7 +66,14 @@ private void onSchemaChange(
       }
       responseObserver.onNext(notification.build());
     } catch (Throwable t) {
-      responseObserver.onError(t);
+      try {
```

Comment:
Should these 2 statements be on separate try-catch blocks, as they could both potentially throw exception?